plugins/kosync.koplugin/main.lua

```diff
@@ -507,6 +542,28 @@ function KOSync:getLastProgress()
     end
 end
 
+function KOSync:getDocumentDigest()
+    if self.kosync_checksum_method == CHECKSUM_METHOD.FILENAME then
+        return self:getFileNameDigest()
+    else
+        return self:getFileDigest()
+    end
+end
+
+function KOSync:getFileDigest()
+    return self.view.document:fastDigest()
+end
+
+function KOSync:getFileNameDigest()
+    local file = self.ui.document.file
+    if not file then return end
+
+    local filePath, fileName = util.splitFilePathName(file)
```

Comment:
```
Luacheck results
Checking plugins/kosync.koplugin/main.lua         1 warning

    plugins/kosync.koplugin/main.lua:561:11: unused variable 'filePath'
```
Just add `-- luacheck: no unused` at the end of line to avoid it.